src/CMakeLists.txt

```diff
@@ -641,13 +648,24 @@ set(MIOPEN_CK_LINK_FLAGS)
 if(MIOPEN_USE_COMPOSABLEKERNEL)
 set(MIOPEN_CK_LINK_FLAGS composable_kernel::device_operations hip::host)
 endif()
-
```

Comment:
Just wondering why did you remove this blank line?

Reply:
[Rationale] @Dmantri98 The less changes we do the less probability of merge conflicts for other developers.

Reply:
Reverted.